### PR TITLE
added sup3r batch utility with cli based on reV batch module

### PR DESCRIPTION
**Pull request by grantbuster**

### CODE REVIEW

File: sup3r/cli.py [hunk 1]
```diff
@@ -289,6 +290,58 @@ def valid_pipeline_keys(ctx):
...
+              'stdout/stderr will not be captured, but you can set a '
+              'pipeline "log_file" to capture logs.')
+@click.option('-v', '--verbose', is_flag=True,
+              help='Flag to turn on debug logging.')
+@click.pass_context
+def batch(ctx, dry_run, cancel, delete, monitor_background, verbose):
+    """Create and run multiple sup3r project directories based on batch
+    permutation logic.
+
+    The sup3r batch module (built on the reV batch functionality) is a way to
```
Comment (bnb32) [inline sup3r/cli.py:313]: Cool idea. What are you using this for?

**Review by bnb32 — COMMENTED** [review]
Worth adding a cli test? Or this is probably taken care of in reV?

File: sup3r/cli.py (hunk 1 above)
Comment (grantbuster) [reply sup3r/cli.py:313]: all sorts of stuff, you can use it to test different models or fwp chunk sizes+overlaps, or to set up jobs for multiple years.

**Comment by grantbuster** [comment]
Yeah i think it's sufficiently tested in reV.

**Review by bnb32 — APPROVED** [review]

File: sup3r/cli.py (hunk 1 above)
Comment (bnb32) [reply sup3r/cli.py:313]: Yeah multiple years is what I though of first. Looks good!

**Pull request merged**